.github/workflows/automationTemplate.yaml

```diff
@@ -75,7 +79,7 @@ jobs:
         with:
           branch: automation/add-new-${{ inputs.platformName }}-versions-${{ env.RELEASE_DATE }}
           title: New ${{ env.PLATFORM_NAME_CAMEL_CASE }} Sdks ${{ env.RELEASE_DATE }}
-          token: ${{ secrets.GH_TOKEN }}
+          token: ${{ secrets.PAT_TEMP }}
```

Comment:
nit: would you mind renaming this secret so that it's a bit more descriptive with regards to what the personal access token is for? `PAT_TEMP` let's me know it's a temporary PAT, but outside of that, I wouldn't be sure what the PAT has access to or is used for without a bit more context, so renaming this secret would be great